packages/common/src/ui/components/buttons/ButtonShowcase.stories.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { FC } from 'react';
+import { Grid, Paper, Typography } from '@mui/material';
+import { Story } from '@storybook/react';
+import { FlatButton } from './FlatButton';
+import { Book } from '../../icons';
+import { BaseButton, ButtonWithIcon } from '.';
+import { Customers } from '../../icons';
+import { DialogButton, IconButton } from '..';
+
+const getOnClick = (someText: string) => () => {
```

Comment:
I really liked what you did with the icons and tried to do similarly..

Reply:
that looks awesome! great idea